assets/dev/scss/editor/_templates-modal.scss

```diff
@@ -300,7 +300,8 @@ label.elementor-template-library-order-label {
 	}
 }
 
-.elementor-template-library-template-remote {
+.elementor-template-library-template-remote,
+.elementor-template-library-template-remote-ehp {
```

Comment:
@nicoladj77 I think we'll need to add this styling in hello-plus, or use another class name